pkg/monitoring/alloy/configmap_test.go

```diff
@@ -44,10 +46,26 @@ func TestGenerateAlloyConfig(t *testing.T) {
 				},
 			},
 			tenants:    []string{"tenant1", "tenant2"},
-			goldenPath: filepath.Join("testdata", "alloy_config_multitenants.river"),
+			goldenPath: filepath.Join("testdata", "alloy_config_multitenants.wc.river"),
+		},
+		{
+			name: "TwoTenantsInMC",
+			cluster: &clusterv1.Cluster{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      managementClusterName,
```

Comment:
To differentiate between a WC and a MC, well, the WC name must match the MC name